dll/win32/feclient/feclient.spec

```diff
@@ -0,0 +1,50 @@
+@ stub EfsUtilGetCurrentKey
```

Comment:
Honestly I don't know how wine could satisfy programs that genuinely use feclient, because the first time they would invoke such an export, this would trigger a stack corruption due to the exports being "whatever" but the program expecting them to be stdcall...